code/modules/mob/living/brain/brain_item.dm

```diff
@@ -60,6 +60,8 @@
 	name = "[L.name]'s brain"
 	if(brainmob || decoy_override)
 		return
+	if(L.mind.current != L)
```

Comment:
Inserting a brain into a body will kick whoever's in the body out, but only if the brain is occupied. This prevents brains from being occupied when they shouldn't be.